### ISSUE TITLE
[Feature Request] Don't show validation errors until validate() is called

### ISSUE THREAD

**Issue by eduardo76** [vuetifyjs/vuetify#3530]
### New Functionality
It will allow form validations happen only when this.$ref.form.validate() is called.
This avoids form errors be displayed when a field loses focus.
A example like this can be seen on google contacts. No errors is displayed when a field loses focus, but when save button is clicked

### Improvements
Forms will be more clean if no errors are displayed.

### Bugs or Edge Cases it Helps Avoid
It avoids user to be worried about errors during filling in the forms

**Comment by nekosaur** [comment]
`lazy-validation` should not change. It's working as intended, validating on blur. This is more akin to manual validation

**Comment by Djaler** [comment]
I'm really looking forward to this
@KaelWD please :D

**Comment by wubzz** [comment]
Additional use-case is when reusing forms by clearing the underlying data model upon saving. For instance I have a `<v-data-table>` with a *footer* slot containing fields tied to a data model using `v-model` on each respective field. When I save the users input I call a `.clear()` method on said model which clears all the values so that the user can input another row. This results in more or less all fields displaying error messages upon a successful save...

My workaround for now is to temporarily empty the `$rules` prop right after a save, which is not optimal.

**Comment by KaelWD** [comment]
@wubzz `$refs.form.resetValidation()`

**Comment by albernhagen** [comment]
I am also interested in this feature.

**Comment by Myolisi** [comment]
Is this feature available already?

**Comment by milost1982** [comment]
What is the status of this feature request? Will it be implemented? This is very useful to have and currently we need it in our projects.

We'd like to validate form and display errors only after validate() is called explicitly (eg. user tries cliks the "submit" button). Showing errors as user is filling the form (either on each input change or blur event) can be distracting and we'd like to postpone that until user tries to submit the form.

**Comment by eduardo76** [comment]
I guess this will not change. My workaround for this is to set rules before this.$refs.form.validate() method call. This works good.

```
save() {
   this.activateRules();
   
    this.$nextTick(() => {
      if (this.$refs.form.validate()) {
        <code here>
        this.rules = {}
      }
    }
},

activateRules() {
    this.rules = {
        email: [
          v => !!v || "E-mail can't be empty",
          v =>
            /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(v) ||
            "Invalid e-mail"
        ],
        password: [
          v => !!v || "Password can't be empty"
          // (v) => v && v.length >= 4 || 'Minimum 4 characters'
        ]
      }
}
```

I hope this helps you.

**Comment by rmirabelle** [comment]
@eduardo76  Brilliant. This works as a hack fix for Vuetify's over-eager form validation and should be the default behavior.  

The "industry standard" validation behavior for forms (for at least the last decade or so) is to **wait to eagerly validate inputs (i.e. on blur) until submit has been attempted**. 

Among other things, Vuetify's default validation behavior makes it impossible to auto-focus the first input of any form.  By simply auto-focusing the input, you're immediately forced into eager/blur-based validation, which means that ANY interaction with ANY other element on the page (including cancel, dismiss dialog, etc) incorrectly triggers validation. 

This is super frustrating.  I'd really like to see Vuetify validation adhere to the standard model.

**Comment by johnleider** [comment]
I'm going to take this over as a small feature for v2.5 while v3 is being worked on. Thank you all for your patience.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by maxanstey** [comment]
In line with what @KaelWD suggested, I simply did:

```
<v-text-field
  :rules="[rules.required]"
  validate-on-blur
  @blur="() => $refs.form.resetValidation()"
/>
```

It feels a bit hacky, but it gets the job done.

**Comment by johnleider** [comment]
As of right now I'm leaning towards making this the default behavior in v3. Thoughts?

**Comment by eduardo76** [comment]
> As of right now I'm leaning towards making this the default behavior in v3. Thoughts?

It would be very nice.

**Comment by johnleider** [comment]
resolved by #14026. All validation in v3 will be lazy by default.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by Djaler** [comment]
Is there any chance to backport it to v2?

**Comment by johnleider** [comment]
Not at this time.
